module/PowerNSX.psm1

```diff
@@ -9111,12 +9111,14 @@ function Get-NsxLogicalSwitch {
     Transport Zone.
 
     .EXAMPLE
+    Get-NsxTransportZone | Get-NsxLogicalswitch -name LS1
 
-    Example1: Get a named Logical Switch
-    PS C:\> Get-NsxTransportZone | Get-NsxLogicalswitch -name LS1
+    Get a named Logical Switch
```

Comment:
incidentally - Get-LogicalSwitch does not require a pipeline input of TZ any more, and defaults to checking in all TZs if not specified.